.github/workflows/tests.yml

```diff
@@ -53,6 +53,10 @@ jobs:
 
   linter_check:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node: ["12", "14", "16"]
```

Comment:
A linter CI is here just to check the style and the linting, not to check it can run with different versions